harness/determined/cli/token.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import json
+from typing import Any, List, Sequence
+
+from determined import cli
+from determined.cli import errors, render
+from determined.common import api, util
+from determined.common.api import authentication, bindings
+
+TOKEN_HEADERS = [
+    "ID",
+    "User ID",
+    "Description",
+    "Created At",
+    "Expires At",
+    "Revoked",
+    "Token Type",
+]
+
+
+def render_token_info(token_info: Sequence[bindings.v1TokenInfo]) -> None:
+    values = [
+        [t.id, t.userId, t.description, t.createdAt, t.expiry, t.revoked, t.tokenType]
+        for t in token_info
+    ]
+    render.tabulate_or_csv(TOKEN_HEADERS, values, False)
+
+
+def describe_token(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    filter_data = json.dumps({"Token_Ids": args.token_id})
```

Comment:
see my comment in the master code. i don't see why we should do this complicated json parsing filter logic. this would be cleaner if we just pass in the "filter" options in the request object.